addon/components/listing.js

```diff
@@ -437,5 +437,5 @@ function hasOrder(subject, store, graph) {
 function getOrder(subject, store, graph) {
   let order = store.any(subject, ORDER, undefined, graph);
 
-  return parseInt(order.value);
+  return parseInt(order?.value) || 0;
```

Comment:
do you feel that 0 is a sane default or do you feel that undefined makes more sense?

Reply:
_haven't tested it on the ember submission form fields dummy app_

Should definitely return a number as I assume that it cannot handle the `null` or `undefined` as an order value.  Is zero the first order number? Never seen a field have order zero before so It needs to be `1`? 

Question: should the component be responsible to add the default value to the tripleStore?

Note: Maybe a strange thought but if you create a listing that has as default one field assigned. And the second field cannot get the order and falls back on the default than the second field will have the default order of `0` (or the `1` depends on what we end up to taking)

Reply:
changed the order to 1 if undefined! i would not add anything to the triplestore in this case. Feels like a higher level thing.